Sources/RediStackPerformanceTester/main.swift

```diff
@@ -0,0 +1,42 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the RediStack open source project
+//
+// Copyright (c) 2023 RediStack project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of RediStack project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import RediStack
+import Foundation
+import NIOCore
+import NIOEmbedded
+import RESP3
+
+func benchmark(label: String = #function, _ n: Int = 100_000, run: () throws -> Void) throws {
```

Comment:
At the time we just wanted to do what NIO does. However the NIO team has moved forward and they now use the `.package(url: "https://github.com/ordo-one/package-benchmark.git", from: "1.11.1"),`, do you think we want to do the same?

https://github.com/apple/swift-nio/tree/main/Benchmarks

wdyt?

Reply:
I love it, I'll update to it